arch/templates/arch/hint_list.html

```diff
@@ -75,3 +75,15 @@ <h2>Hint {{ hint.number }}%</h2>
     </a>
   </div>
 {% endblock layout-content %}
+
```

Comment:
Scripts need to be enclosed in `{% block scripts %}` ... `{% endblock scripts %}` to show up.